projects/adrv9009/common/adrv9009_qsys.tcl

```diff
@@ -1,3 +1,28 @@
+
+# TX parameters
+set TX_NUM_OF_LANES 4      ; # L
+set TX_NUM_OF_CONVERTERS 4 ; # M
+set TX_SAMPLE_WIDTH 16     ; # N/NP
+
+set TX_SAMPLES_PER_CHANNEL [expr $TX_NUM_OF_LANES * 32 / \
+                                ($TX_NUM_OF_CONVERTERS * $TX_SAMPLE_WIDTH)] ; # L * 32 / (M * N)
+
+# RX parameters
+set RX_NUM_OF_LANES 2      ; # L
+set RX_NUM_OF_CONVERTERS 4 ; # M
+set RX_SAMPLE_WIDTH 16     ; # N/NP
+
+set RX_SAMPLES_PER_CHANNEL [expr $RX_NUM_OF_LANES * 32 / \
+                                ($RX_NUM_OF_CONVERTERS * $RX_SAMPLE_WIDTH)] ; # L * 32 / (M * N)
+
+# RX Observation parameters
+set RX_OS_NUM_OF_LANES 2      ; # L
+set RX_OS_NUM_OF_CONVERTERS 4 ; # M
```

Comment:
We should move the default ORx setting to M=2

Reply:
Fixed. Thanks!